src/components/CardDescription/CardDescriptrion.js

```diff
@@ -11,10 +11,10 @@ class CardDescription extends Component {
 
   /** Toggle editing and input text */
   toggle = () => {
-    this.setState({
+    this.setState(prevState => ({
       editing: !this.state.editing,
```

Comment:
Du verwendest wieder `this.state.editing` statt `prevState.editing`. Das falsch :b